containers/EmpState.ts

```diff
@@ -50,83 +50,84 @@ const initState = {
   finderAddress: null,
   expiryPrice: null,
 };
+// get state from EMP
+export const queryState = async (emp: any) => {
```

Comment:
this was a pure function which I exported in case it could be useful somewhere else

Reply:
Won't `expiryPrice` fail on a Perp? Or is the assumption that `queryState` would never be called on a Perp

Reply:
yea this currently is not called with ther perp because there are incompatibilities. This is why the general info section in the screenshot is currently empty.